app/components/user-profile-bio.hbs

```diff
@@ -1,428 +1,450 @@
-      class="bigcancel"
-      {{action (perform cancel)}}
-    >
-      <FaIcon @icon="undo" />
-    </button>
-  {{else if isManageable}}
-    <button type="button" class="manage" {{action (perform manage)}}>
-      <FaIcon @icon="edit" />
-    </button>
-  {{/if}}
-</div>
-{{#if (await usernameMissing)}}
-  <div class="error">
-    {{t "general.missingRequiredUsername"}}
-  </div>
-{{/if}}
-<p class="primary-school">
-  <strong>
-    {{t "general.primarySchool"}}:
-  </strong>
-  {{get (await user.school) "title"}}
-</p>
-<div class="form">
-  <div
-    class="item
-      {{if (includes "firstName" updatedFieldsFromSync) "synced-from-directory"
-      }}"
-  >
-    <label>
-      {{t "general.firstName"}}:
-    </label>
-    {{#if isManaging}}
-      <input
-        type="text"
-        value={{firstName}}
-        oninput={{action (mut firstName) value="target.value"}}
-        onkeyup={{action "addErrorDisplayFor" "firstName"}}
-        data-test-first-name-input
-      >
-      {{#if
-        (and
-          (v-get this "firstName" "isInvalid") (includes "firstName" showErrorsFor)
-        )
-      }}
-        <span class="message error">
-          {{v-get this "firstName" "message"}}
-        </span>
-      {{/if}}
-    {{else}}
-      <span class="value">
-        {{@user.firstName}}
-      </span>
-    {{/if}}
-  </div>
-  <div class="item">
-    <label>
-      {{t "general.middleName"}}:
-    </label>
-    {{#if isManaging}}
-      <input
-        type="text"
-        value={{middleName}}
-        oninput={{action (mut middleName) value="target.value"}}
-        onkeyup={{action "addErrorDisplayFor" "middleName"}}
-        data-test-middle-name-input
-      >
-      {{#if
-        (and
-          (v-get this "middleName" "isInvalid")
-          (includes "middleName" showErrorsFor)
-        )
-      }}
-        <span class="message error">
-          {{v-get this "middleName" "message"}}
-        </span>
-      {{/if}}
-    {{else}}
-      <span class="value">
-        {{@user.middleName}}
-      </span>
-    {{/if}}
-  </div>
-  <div
-    class="item
-      {{if (includes "lastName" updatedFieldsFromSync) "synced-from-directory"
-      }}"
-  >
-    <label>
-      {{t "general.lastName"}}:
-    </label>
-    {{#if isManaging}}
-      <input
-        type="text"
-        value={{lastName}}
-        oninput={{action (mut lastName) value="target.value"}}
-        onkeyup={{action "addErrorDisplayFor" "lastName"}}
-        data-test-last-name-input
-      >
-      {{#if
-        (and
-          (v-get this "lastName" "isInvalid") (includes "lastName" showErrorsFor)
-        )
-      }}
-        <span class="message error">
-          {{v-get this "lastName" "message"}}
-        </span>
-      {{/if}}
+<div
+  data-test-user-profile-bio
+  class="user-profile-bio small-component {{if this.hasRecentlySaved "has-saved" "has-not-saved"}}"
+  {{did-insert (perform this.load)}}
+  {{did-update (perform this.load) @user}}
```

Comment:
Since the `load` action depends on `@isManaging` it should be added here as well.

Reply:
:heavy_check_mark: